pkg/helm/helm.go

```diff
@@ -191,11 +192,11 @@ func (h *Hydrator) getChartVersion(ctx context.Context) error {
 func fetchNewToken(ctx context.Context) (*oauth2.Token, error) {
 	creds, err := google.FindDefaultCredentials(ctx, "https://www.googleapis.com/auth/cloud-platform")
 	if err != nil {
-		return nil, fmt.Errorf("failed to find default credentials: %w", err)
+		return nil, errors.Wrap(err, "failed to find default credentials")
```

Comment:
Just curious why changing to `github.com/pkg/errors`? IIRC, you had a comment before for this archived and random library and would prefer to use `fmt.Errorf`.

Reply:
I don't like the github.com/pkg/errors library, but I do like consistency, and we're already using this library throughout Config Sync. So I added it here for consistency. If we want to stop using it, someone will need to do a very large refactor.

I didn't add it to all of the errors, because some of them are printing the error first and then the stdout/stderr, which isn't compatible. But that kind of error is also problematic when logged, due to the use to syslog, which interprets each line as its own log message. So all that needs to be fixed at some point later anyway.

Reply:
Reverted changes to the errors.